db/seeds.rb

```diff
@@ -12,3 +12,4 @@
 sources_path = ENV['TERRAFORM_SOURCES_PATH']
 sources_path ||= Rails.root.join('vendor', 'sources')
 Source.import_dir(sources_path, validate: false)
+Terraform.new.validate(true)
```

Comment:
somehow this conceptually feels strange to be place here. but maybe there is no better place. I'm just wondering and looking at best practices regarding rails.

Reply:
This is just to ensure that imported Terraform sources are validated all in a shot. We could have just removed out the `validate: false` in the line above, but then the validation would occur for each file. This way we instead skip it at first, and then run validate for all of them at once.

Reply:
my concern here is more like should be the validation placed when acessing the `/` handler or in db? 

I was more in doubt on this